### ISSUE TITLE
Can't vein mine trees

### ISSUE THREAD

**Issue by TimWerdin** [bloop428/Excavation#17]
When I press the vein mining key and try to vein mine a oak tree with a iron axe I only mine the one block that I was hitting.
Mining ores works fine.

Details:
forge-32.0.75
excavation-1.5

excavation-common.toml:
```

[client]
	#Disable the keybind and activate excavation by crouching.
	crouch = false

[server]
	#All blocks are placed in the player's inventory unless it is full (False [Will cause extra lag])
	vacuum = false
	#Maximum blocks to break with one excavation (Integer)
	#Range: > 1
	max = 64
	#Multiply the default Minecraft block exhaustion by this much (Double)
	#Range: 0.0 ~ 1.7976931348623157E308
	exhaustion = 1.0
	#Does the player need to mine with a valid tool?
	tool = true
```

excavation-groups.json:
```
[["minecraft:oak_log"]]
```

excavation-server.toml
```

["General Category"]
	#All blocks are spawned under the player (0 false, 1 true)
	#Range: 0 ~ 1
	vacuum = 0
	#Maximum blocks to break with one excavation (Integer)
	#Range: > 1
	max = 64
	#Multiply the default Minecraft block exhaustion by this much (Double)
	#Range: 0.0 ~ 1.7976931348623157E308
	exhaustion = 1.0
	#Does the player need to mine with a valid tool? (0 false, 1 true)
	#Range: 0 ~ 1
	tool = 1
```

**Comment by bloop428** [comment]
The server config still exists? It shouldnt. Did you use the mod before 1.5?

**Comment by TimWerdin** [comment]
I have used the mod before 1.5. As you have stated, that the server config shouldn't exists any more I deleted it.
I also updated to forge-32.0.108 which did not fix the issue.

After disabling tool mode I was able to vein mine a tree. With tool mode enabled I can't vein mine a tree. And yes I am using an axe. To be precise an iron axe.
I also saw that vein mining nether quartz with an iron pickaxe was not possible while the tool mode was enabled.
